src/include/type/value.h

```diff
@@ -169,6 +169,9 @@ class Value : public Printable {
   // Is a value null?
   inline bool IsNull() const { return size_.len == PELOTON_VALUE_NULL; }
 
+  // Is a value not null?
+  inline bool IsNotNull() const { return !IsNull();}

```

Comment:
Why do we actually want this? I think this sets a bad precedent on what functions we define for our classes.

I would remove this.

Reply:
OK, I will remove it. This is because for the llvm part, we have IsNull() and IsNotNull() so that we don't bring in parser-related objects into the expression. And I have IsNotNull() here just to ensure the normal part is in consistency with the llvm one.